README.md

```diff
@@ -93,18 +94,17 @@ fully-patched installation of MOULa provided by Cyan Worlds.
 1. Copy the **files from your existing MOULa installation** to a new folder, or
    install fresh if you do not already have it installed. This folder will be
    referred to as *MOUL-OS* for the remainder of these instructions.
-2. Copy the **Python and SDL** folders from `<build_dir>\bin` into the *MOUL-OS*
-   folder, as well as the **files in dat** into the existing dat folder.
+2. Copy the **client** folders from `<build_dir>\install` into the *MOUL-OS*
+   folder.
 3. Copy the example_server.ini file from the root of the Plasma repository into
    your *MOUL-OS* folder, and rename it as **server.ini**. If you are running
    your own dirtsand server or are connecting to one run by someone else, use
    the server.ini generated from that.
 4. Copy the **DLLs** from the development libraries bundle, as well as the DLLs
    PhysXLoader.dll, NxExtensions.dll, NxCooking.dll, and NxCharacter.dll from
    your PhysX SDK installation into the *MOUL-OS* folder.
-5. Copy the **resource.dat** file from `<build_dir>\bin` to the *MOUL-OS* folder,
-   or from http://www.guildofwriters.org/tools/resource.dat if you did not build
-   your own.
+5. Download and copy http://www.guildofwriters.org/tools/resource.dat into
```

Comment:
Why remove the instructions for using the locally-built copy?

Reply:
`resource.dat` will only be generated if you manually `pip install -r requirements.txt` or otherwise have the correct packages preinstalled. I figure that's a problem for another PR :smile: